ivy/array/elementwise.py

```diff
@@ -1381,6 +1381,27 @@ def log2(self: ivy.Array, *, out: Optional[ivy.Array] = None) -> ivy.Array:
             an array containing the evaluated base ``2`` logarithm for each element
             in ``self``. The returned array must have a real-valued floating-point
             data type determined by :ref:`type-promotion`.
+
+        Examples
+        --------
+        Using :code:`ivy.Array` instance method:
```

Comment:
We're using `class` instead of code as it is `ivy.Array` class
```suggestion
        Using :class:`ivy.Array` instance method:
```